admin-cli/README-cli.md

```diff
@@ -0,0 +1,52 @@
+# admin-cli
+
+This is a command line tool for catalog admins to examine the contents of the catalog
+database.
+
+To use it:
+
+1. Build:
+```shell
+./gradlew :admin-cli:bootJar
+```
+
+2. Start the database proxy. `ENV` is the database environment to use. Note that you currently
+must use port `5432`, which may conflict with a locally running postgres. If the port is
+in use, `db_connect.sh` will report an error and exit.
+
+```shell
+ENV=dev PORT=5432 ./scripts/db_connect.sh
+```
+
+In the proxy output, look for the database name, username and password.
+
+3. Run the command:
+
+```shell
+DATABASE_NAME=catalog DATABASE_USER=catalog DATABASE_USER_PASSWORD=<password> \
+  java -jar admin-cli/build/libs/admin-cli-<version>.jar validate
+```
+
+You can also run the command without building a jar using `bootRun` and `--args`:
+
+```shell
+DATABASE_NAME=catalog DATABASE_USER=catalog DATABASE_USER_PASSWORD=<password> \
+  ./gradlew :admin-cli:bootRun --args=validate
+```
+
+## Commands supported
```

Comment:
JMO but I could see this getting out of date as (hopefully) additional commands are supported.  Had you considered putting these inside a CLI `--help` option?

Reply:
I agree, it'd be better to rely on the tool itself to document its usage. If an incorrect command is entered you do get a short "usage" message:
```
$ java -jar admin-cli.jar xyzzy
unknown command xyzzy. Valid commands are: 'list, get, validate'
```

So there is the start of some self-documentation. As things get more complicated this help will need to improve, but I think it can wait until then.